src/site/content/en/media/video-and-source-tags/index.md

```diff
@@ -0,0 +1,361 @@
+---
+layout: post
+title: The <video> and <source> tags
+authors:
+  - samdutton
+  - joemedley
+description: |
+  You've properly prepared a video file for the web. You've given it correct
+  dimensions and the correct resolution. You've even created separate WebM and
+  MP4 files for different browsers. For anyone to see it, you still need to add
+  it to a web page.
+date: 2014-14-15
+updated: 2020-08-20
+---
+
+You've properly [prepared a video file](../prepare-media/) for the web. You've
+given it correct dimensions and the correct resolution. You've even created
+separate WebM and MP4 files for different browsers.
+
+For anyone to see it, you still need to add it to a web page. Doing so properly
+requires two HTML elements: the
+[`<video>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video)
+element and the
+[`<source>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source).
+In addition to basics about these tags, this section also explains attributes
+you should add to those tags to craft a good user experience.
+
+{% Aside %}
+You always have the option of uploading your file to
+[YouTube](https://www.youtube.com/) or [Vimeo](https://vimeo.com/). In many
+cases, this is preferable to the procedure described here. Those services handle
+formatting and filetype conversion for you, as well as provide the means to
+embedd a video in your web page. If need to manage this yourself, read on.
+{% endAside %}
+
+## Specify a single file
+
+Although it's not recommended, you can use the video element by itself. Always
+use the `type` attribute as shown below. The browser uses this to determine if
+it can play the provided video file. If it can't, the enclosed text displays.
+
+```html
+<video src="chrome.webm" type="video/webm">
+    <p>Your browser cannot play the provided video file.</p>
+</video>
+```
+
+### Specify multiple file formats
+
+Recall from [File basics](../file-basics) that not all browsers support the same
+video formats. The `<source>` element lets you specify multiple formats as a
+fallback in case the user's browser doesn't support one of them.
+
+The example below produces the ebedded video that immediately follows.
+
+```html
+<video controls>
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.webm" type="video/webm">
+  <source src="https://storage.googleapis.com/web-dev-assets/video-and-source-tags/chrome.mp4" type="video/mp4">
+    <p>Your browser cannot play the provided video file.</p>
```

Comment:
```suggestion
  <p>Your browser cannot play the provided video file.</p>
```
Fixing the indentation since it's a sibling element of the `source` elements